package.json

```diff
@@ -87,7 +102,7 @@
       "eslint --ext .ts"
     ]
   },
-  "packageManager": "pnpm@6.32.11",
+  "packageManager": "pnpm@7.0.1",
```

Comment:
for corepack to work, is this needed in workspace root only or in all package.json files in the repo?
https://github.com/nodejs/corepack

Reply:
From my experience, only the top-level package.json is required (not well tested tho)